res/controllers/Denon-MC7000-scripts.js

```diff
@@ -496,7 +496,12 @@ MC7000.PadButtons = function(channel, control, value, status, group) {
                 engine.setValue(group, "hotcue_" + i + "_activate", true);
             } else if (control === 0x14 + i - 1 && value === 0x00) {
                 engine.setValue(group, "hotcue_" + i + "_activate", false);
-                engine.setValue(group, "slip_enabled", 0);
+                if (engine.getValue(group, "slip_enabled")) {
+                    engine.setValue(group, "slip_enabled", false);
+                    engine.beginTimer(250, function() {
+                        engine.setValue(group, "slip_enabled", true);
+                    }, true);
+                }
```

Comment:
make sure to document this in the manual

Reply:
I gonna update documentation when this PR is open. There are still a few ideas I try to put into place.